third_party/amd/backend/driver.py

```diff
@@ -165,7 +141,7 @@ def __init__(self):
         # This way we don't need to escape-quote C code curly brackets and we can replace
         # exactly once.
         src = src.replace('/*py_libhip_search_path*/', libhip_path, 1)
-        mod = compile_module_from_src(src, "hip_utils")
+        mod = compile_module_from_src(src, "hip_utils", include_dirs=include_dirs)
```

Comment:
Nit: maybe follow nvidia side to be explicit about the keys for these values too?